### PR TITLE
[MM-54]: Added the test cases for revoking the application on gitlab and proper message is displayed for member access request in todo.

### PR DESCRIPTION
**Pull request by arush-vashishtha**
### Summary
The following PR consists the test cases for the scenario,

- Revoking the application on Gitlab.
- Display proper message for member access request in todo.

### CODE REVIEW

File: data/test-cases/plugins/gitlab/sidebar/merge_request.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+**Step 2**
+
+1. Accept or reject any desired member access request on Gitlab.
+2. Navigate to MM and click on the `To-do list` in the LHS menu or enter `/gitlab todo` in the channel or DM/GM on MM.
+3. Check the desired member access request in the RHS of `To-do list` on MM.
+
+**Expected**
+
+The member access request to a project/group should show the description of the request in proper UI on MM.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/gitlab/sidebar/merge_request.md]: in RHS

File: data/test-cases/plugins/gitlab/sidebar/refresh_sidebar_after_revoke.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Revoke the OAuth token for your app from the connectd gitlab account.
+2. Navigate to MM and click on the refresh button in the LHS of gitlab.
+3. Open the gitlab bot chat.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/gitlab/sidebar/refresh_sidebar_after_revoke.md]: DM from Gitlab bot

File: data/test-cases/plugins/gitlab/sidebar/refresh_sidebar_after_revoke.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+**Step 3**
+
+1. Revoke the OAuth token for your app from the connectd gitlab account.
+2. Navigate to MM and enter any slash command that fetches the data from the gitlab.
+3. Open the DM from Gitlab bot.
+
+**Expected**
+
+The LHS of Gitlab should be removed and the user should get a message in the DM from Gitlab bot that the account has been disconnected on MM.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/gitlab/sidebar/refresh_sidebar_after_revoke.md]: Should not it be that action buttons should get removed, complete LHS does not get removed as there is still a connection icon.

Comment (arush-vashishtha) [reply data/test-cases/plugins/gitlab/sidebar/refresh_sidebar_after_revoke.md]: As discussed with Raghav and Kshitij, they asked to change it to `All the Gitlab icons in the LHS should be removed and the user should get a message in the DM from Gitlab bot that the account has been disconnected on MM.`

**Review by AayushChaudhary0001 — APPROVED** [review]

File: data/test-cases/plugins/gitlab/sidebar/merge_request.md
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/gitlab/sidebar/merge_request.md:1]: According to the PR description this file should not be here

File: data/test-cases/plugins/gitlab/sidebar/refresh_sidebar_after_revoke.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Revoke the OAuth token for your app from the connectd gitlab account.
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/gitlab/sidebar/refresh_sidebar_after_revoke.md]:
```suggestion
1. Revoke the OAuth token for your app from the connected gitlab account.
```

File: data/test-cases/plugins/gitlab/sidebar/refresh_sidebar_after_revoke.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+**Step 3**
+
+1. Revoke the OAuth token for your app from the connectd gitlab account.
+2. Navigate to MM and enter any slash command that fetches the data from the gitlab.
+3. Open the DM from Gitlab bot.
+
+**Expected**
+
+All the Gitlab icons in the LHS should be removed and the user should get a message in the DM from Gitlab bot that the account has been disconnected on MM.
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/gitlab/sidebar/refresh_sidebar_after_revoke.md]:
```suggestion
All the Gitlab icons in the LHS should be removed and the user should get a message in the DM from Gitlab bot that their account has been disconnected on MM.
```

Comment (raghavaggarwal2308) [reply data/test-cases/plugins/gitlab/sidebar/refresh_sidebar_after_revoke.md]: Fix everywhere

**Review by raghavaggarwal2308 — CHANGES_REQUESTED** [review]

File: data/test-cases/plugins/gitlab/sidebar/merge_request.md
Comment (arush-vashishtha) [reply data/test-cases/plugins/gitlab/sidebar/merge_request.md:1]: updated.

File: data/test-cases/plugins/gitlab/sidebar/merge_request.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Generate a access request to your desired private project/group from another user on Gitlab.
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/gitlab/sidebar/merge_request.md]:
```suggestion
1. Generate an access request to your desired project/group from another user on Gitlab.
```

File: data/test-cases/plugins/gitlab/sidebar/merge_request.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+**Step 2**
+
+1. Accept or reject any desired member access request on Gitlab.
+2. Navigate to MM and click on the `To-do list` icon in the LHS menu on MM.
+3. Check the desired member access request in the RHS of `To-do list` on MM.
+
+**Expected**
+
+The member access request to a project/group should show the description of the request in proper UI in RHS of Gitlab on MM.
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/gitlab/sidebar/merge_request.md]:
```suggestion
The member access request to the project/group should show the description of the request in proper UI in RHS of Gitlab on MM.
```

**Review by raghavaggarwal2308 — CHANGES_REQUESTED** [review]

**Review by raghavaggarwal2308 — APPROVED** [review]

**Pull request merged**